components/basket/BasketProductList.tsx

```
@@ -19,47 +33,61 @@ const BasketProductList = () => {
-      <View>
-        <BasketProduct />
+      <View className="grow">
+        {shoppingSession &&
+          shoppingSession?.length > 0 &&
+          shoppingSession?.[0]?.cartItems.map((cartItem) => (
+            <BasketProduct
+              cartItem={cartItem}
+              productQuantities={productQuantities}
+              handleIncreaseQuantity={handleIncreaseQuantity}
+              handleDecreaseQuantity={handleDecreaseQuantity}
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)